pyGHDL/lsp/lsp.py

```diff
@@ -118,15 +118,12 @@ def handle(self, msg):
             try:
                 response = fmethod(**params)
             except Exception as e:
-                log.exception(
-                    "Caught exception while handling %s with params %s:", method, params
-                )
+                log.exception("Caught exception while handling %s with params %s:", method, params)
                 self.show_message(
                     MessageType.Error,
-                    (
-                        "Caught exception while handling {}, "
-                        + "see VHDL language server output for details."
-                    ).format(method),
+                    ("Caught exception while handling {}, " + "see VHDL language server output for details.").format(
```

Comment:
Regardless of line length, the `+` here is unnecessary.

Reply:
See https://github.com/ghdl/ghdl/commit/11f05af9cbfc36ab24c9f08e0a62e198be7b2e9e. It seems that black does not break those long strings, so the 88 char width is not respected. I guess @tgingold might have used `+` to work around that.